### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by DevBennyturtle**
Fixed startup.py and allowed parameters to be forwarded to Server.py
having issues with something calling startup.py after launch
I think it has always been happening but there were no print statements in main.py so we never noticed it

### CODE REVIEW

**Comment by hydrostaticcog** [comment]
This needs to target the develop branch

**Comment by Geolykt** [comment]
This PR is conflicting with #53 , I would wait until that one is released to build you changes ontop of that one or PR into that PR.

File: classes/Server.py [hunk 1]
```diff
@@ -1,110 +1,109 @@
...
-        if reactor.running:
-            reactor.stop()
-        try:
-            time.sleep(2)  # Waits for all workers to shut down (there's gotta be a better way)
-        except:
-            pass
-
-    def stop_internal_tick(self):
-        logging.info('Stopping ticks')
+import logging
```
Comment (Geolykt) [inline classes/Server.py:1]: You did something to this file that I don't like.
 I recommend you to look at the line endings and text encoding again.

We only support UTF-8 as of now and I got no idea what the line endings are.
Of course we are up to debate to change something like this

File: classes/mcPy/McPy.py [hunk 2]
```diff
@@ -1,61 +1,67 @@
...
-    parser = Parser()
-
-    if parser.debug:
-        print('Debug mode enabled. Don\'t forget to remove debug flag for maximum performance !')
-    logging_level = logging.DEBUG if parser.debug else logging.INFO
-    logging.basicConfig(format="[%(asctime)s - %(levelname)s - %(threadName)s] %(message)s")
-    logging.root.setLevel(logging_level)
-
-    _launch(parser)
+import logging
```
Comment (Geolykt) [inline classes/mcPy/McPy.py:1]: Same here

File: startup.py [hunk 3]
```diff
@@ -1,175 +1,188 @@
...
-    os.remove("mcPyTempFile.zip")
-    logging.info("To run McPy, rerun this script!")
-else:
-    logging.info("McPy found! Running it...")
-    data = [sys.executable, 'McPy/main.py']
-    if parsedArgs.debug:
-        data.append('--debug')
-    mcpyProcess = subprocess.check_call(data)
-    logging.info("startup.py: Server closed")
+# coding=utf-8
```
Comment (Geolykt) [inline startup.py:1]: Same here

**Review by Geolykt — CHANGES_REQUESTED** [review]
Bloated diff, cannot review properly

**Review by ntoskrnl4 — CHANGES_REQUESTED** [review]
Check your line endings in your IDE. As is, all of the line endings are different, which makes this impossible to review or accept.

**Comment by hydrostaticcog** [comment]
Author left project

**Comment by ntoskrnl4** [comment]
more specifically, they ~~never~~ haven't yet fixed the problems with the PR

**Comment by hydrostaticcog** [comment]
Correct, its a combination of both. Author left project, therefore cannot fix issues.

**Pull request closed without merging**